### PR TITLE
add examples to protos for api docs

### PR DESCRIPTION
**Pull request by vjeffrey**
### :nut_and_bolt: Description: What code changed, and why?
Added examples to the protos we've already defined. Also added a few more details to a few of em

### :athletic_shoe: How to Build and Test the Change
- check out this branch
- run make serve from the automate-chef-io directory
- visit http://localhost:1313/docs/api

### CODE REVIEW

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 1]
```diff
@@ -47,9 +56,11 @@ service ReportingService {
 	/*
 	List report IDs
 
-	List all report IDs optionally using filters. Supports filtering, but not pagination or sorting.
+	List all IDs for the latest report of each node optionally using filters.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	List all IDs for the latest report for each node, with the option of filtering. 
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 2]
```diff
@@ -69,7 +80,9 @@ service ReportingService {
 	/*
 	List controls
 
-	Lists controls from the last run, with the option of applying filters. Supports filtering, but not pagination or sorting.
+	Lists controls from the last run, with the option of applying filters.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	Lists controls from the last run, with the option of filtering. 
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 3]
```diff
@@ -152,12 +177,9 @@ service ReportingService {
 	/*
 	List profiles
 
-	List all profiles in use, with the option of applying filters. Adding a profile filter returns a list of all the nodes using that profile. Supports pagination, filtering, and sorting.
-
-	| Sort paramter | Sort value |
-	| --- | --- |
-	| name | name.lower |
-	| title (default) | title.lower |
+	List all profiles in use, with the option of applying filters.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	List all profiles in use, with the option of filtering. 
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 4]
```diff
@@ -200,8 +223,9 @@ service ReportingService {
 	List nodes
 
 	List all nodes optionally using filters. Supports pagination, filtering, and sorting.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	List all nodes, with the option of filtering. Supports pagination, filtering, and sorting.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 5]
```diff
@@ -375,7 +412,7 @@ message ReportIds {
 message Report {
 	// A unique report identifier.
 	string id = 1;
-	// The id of the node making the report.
+	// The ID of the node making the report.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The reporting node's unique ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 6]
```diff
@@ -375,7 +412,7 @@ message ReportIds {
...
 	string node_id = 2;
 	// The name of the node making the report.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The reporting node name.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 7]
```diff
@@ -395,7 +432,7 @@ message Report {
 	Statistics statistics = 10;
 	// The profiles run as part of this report.
 	repeated Profile profiles = 11;
-	// The id of the compliance scan job associated with the report.
+	// The ID of the compliance scan job associated with the report.
 	string job_id = 12;
 	// The IP address of the node making the report.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The reporting node IP address.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 8]
```diff
@@ -637,7 +674,7 @@ message ProfileMin {
 	string name = 1;
 	// The title of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The profile title.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 9]
```diff
@@ -637,7 +674,7 @@ message ProfileMin {
...
 	string title = 2;
-	// The id of the profile.
+	// The ID of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The profile ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 10]
```diff
@@ -646,7 +683,7 @@ message ProfileMin {
 }
 
 message Node {
-	// The id of this node.
+	// The ID of this node.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The node ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 11]
```diff
@@ -684,7 +721,7 @@ message Kv {
 
 // A summary of the information contained in the latest report for this node.
 message LatestReportSummary {
-	// The id of the latest report.
+	// The ID of the latest report.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The latest report ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 12]
```diff
@@ -699,7 +736,7 @@ message ProfileMeta {
 	string name = 1;
 	// The version of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The profile version.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 13]
```diff
@@ -699,7 +736,7 @@ message ProfileMeta {
...
 	string version = 2;
-	// The unique id of the profile.
+	// The unique ID of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The profile unique ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 14]
```diff
@@ -269,7 +306,7 @@ message ControlItemRequest {
 }
 
 message ControlItem {
-	// The unique id of this control.
+	// The unique ID of this control.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The control's unique ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 15]
```diff
@@ -269,7 +306,7 @@ message ControlItemRequest {
...
 	string id = 1;
 	// The compact description of the control.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The control's compact description.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto (hunk 5 above)
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The reporting node ID.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 16]
```diff
@@ -395,7 +432,7 @@ message Report {
 	Statistics statistics = 10;
 	// The profiles run as part of this report.
 	repeated Profile profiles = 11;
-	// The id of the compliance scan job associated with the report.
+	// The ID of the compliance scan job associated with the report.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The compliance scan job ID associated with the report.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 17]
```diff
@@ -637,7 +674,7 @@ message ProfileMin {
 	string name = 1;
 	// The title of the profile.
 	string title = 2;
-	// The id of the profile.
+	// The ID of the profile.
 	string id = 3;
 	// The version of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
	// The profile version.
```

File: components/automate-gateway/api/compliance/reporting/reporting.proto [hunk 18]
```diff
@@ -684,7 +721,7 @@ message Kv {
 
 // A summary of the information contained in the latest report for this node.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/reporting.proto]:
```suggestion
// A summary of the latest report for this node.
```

File: components/automate-gateway/api/compliance/reporting/stats/stats.proto [hunk 19]
```diff
@@ -236,7 +284,7 @@ message Profile {
 message ProfileList {
 	// Name of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/stats/stats.proto]:
```suggestion
	// The profile name.
```

File: components/automate-gateway/api/compliance/reporting/stats/stats.proto [hunk 20]
```diff
@@ -236,7 +284,7 @@ message Profile {
...
 	string name = 1;
-	// SHA id of the profile.
+	// SHA ID of the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/stats/stats.proto]:
```suggestion
	// The profile SHA ID.
```

File: components/automate-gateway/api/compliance/reporting/stats/stats.proto [hunk 21]
```diff
@@ -236,7 +284,7 @@ message Profile {
...
 	string id = 2;
 	// Total number of failed nodes that executed the profile.
```
Comment (kagarmoe) [inline components/automate-gateway/api/compliance/reporting/stats/stats.proto]: Is this the same as "the total number of times this profile failed"? Or does it include nodes that ran the profile successfully, but might have failed on other profiles?" (I forget, is node:profile 1:1?)

Comment (vjeffrey) [reply components/automate-gateway/api/compliance/reporting/stats/stats.proto]: 1 node: many profiles

i...think it's "the total number of times this profile failed"...but i'm not 100% on that. i'll double check and follow up

1.

Comment (vjeffrey) [reply components/automate-gateway/api/compliance/reporting/stats/stats.proto]: ok ya it's the total number of times this profile failed

Comment (kagarmoe) [reply components/automate-gateway/api/compliance/reporting/stats/stats.proto]: `total number of nodes that failed this profile` -- more active than the profile seeming to fail on its own.

Comment (vjeffrey) [reply components/automate-gateway/api/compliance/reporting/stats/stats.proto]: i'm gonna merge this one first so i'll get that change in the nodes proto one

**Pull request merged**